mi/confs/deployment.toml

```diff
@@ -249,22 +249,28 @@ service_catalog_password = {{ .Values.wso2.config.serviceCatalog.password | quot
 {{- end }}
 {{- end }}
 
-{{ if .Values.wso2.config.dashboard }}
+{{ if .Values.wso2.config.icp }}
 [dashboard_config]
-dashboard_url = {{ .Values.wso2.config.dashboard.url | quote }}
-{{- if .Values.wso2.config.dashboard.heartbeatInterval }}
-heartbeat_interval = {{ .Values.wso2.config.dashboard.heartbeatInterval }}
+icp_url = {{ .Values.wso2.config.icp.url | quote }}
```

Comment:
We shouldn’t modify this since it’s a Micro Integrator configuration.

Reply:
fixed the config key by https://github.com/wso2/helm-mi/pull/17/commits/5f5323e770405df55618a48624c45add90ae140b